checkov/terraform/graph_builder/variable_rendering/evaluate_terraform.py

```diff
@@ -23,9 +26,15 @@
 DIRECTIVE_EXPR = r"\%\{([^\}]*)\}"
 
 COMPARE_REGEX = re.compile(r"^(?P<a>.+)(?P<operator>==|!=|>=|>|<=|<|&&|\|\|)+(?P<b>.+)$")
+CHECKOV_RENDER_MAX_LEN = force_int(os.getenv("CHECKOV_RENDER_MAX_LEN", "10000"))
 
 
 def evaluate_terraform(input_str: str, keep_interpolations: bool = True) -> Any:
+    if CHECKOV_RENDER_MAX_LEN and 0 < CHECKOV_RENDER_MAX_LEN < len(input_str):
```

Comment:
```suggestion
    if isinstance(input_str, str) and CHECKOV_RENDER_MAX_LEN and 0 < CHECKOV_RENDER_MAX_LEN < len(input_str):
```
maybe?

Reply:
It's practically called `input_str`, and it's of type `str` (see the line above, line 32)

Reply:
Just saw the tests 🤦